### ISSUE TITLE
Websocket "send" does not replace inline variables

### ISSUE THREAD

**Issue by skxu** [shoreditch-ops/artillery#112]
Example:

```
{
  "config": {
    // ... other config here ...

    "variables": {
      "zipCode": ["35801", "99501", "85001", "94203", "90210"]
    }
  },
  "scenarios": [
    {
      "engine": "ws",
      "flow": [
        {"send": "{\"zip\":\"{{ zipCode }}\""}}
      ]
    }
  ]
}
```

This will send {"zip": "{{ zipCode }}"}

**Comment by baseprime** [comment]
+1, I figured Artillery could do this, but I guess not. Anyone have any ideas other than creating a new Artillery JSON file on the fly?

**Comment by hassy** [comment]
@skxu @baseprime fixed now and will be available in the upcoming `v1.5.0`
